tools/wallet/src/core/wallet_library.rs

```diff
@@ -12,8 +12,6 @@
 //! https://github.com/rust-bitcoin/rust-wallet/blob/master/wallet/src/walletlibrary.rs
 
 use super::{
```

Comment:
I'm unclear if this is the idiomatic way to do these imports. Please check.

Reply:
Reorganized all imports in the entire module.